src/app/classes/diagram/GraphTrace.ts

```diff
@@ -0,0 +1,25 @@
+import { Element } from './element';
+import { GraphEvent } from './GraphEvent';
+
+
+export class GraphTrace {
+    public get caseIds(): Array<Number> {
+        return this._caseIds;
```

Comment:
Wofür werden die CaseIDs hier verwendet? Sollen die in einer zukünftigen Version noch irgendwie dargestellt werden?

Reply:
Ja genau für eine zukünftige Version. Ich könnte mir vorstellen das wir wie in Prom auch ein Fenster aufgehen lassen wenn man einen Trace selektiert und dann alle einzelnen CaseId's / Traces dargestellt werden. Siehe
https://eventlogs.atlassian.net/browse/EL-38?atlOrigin=eyJpIjoiZTg4ZGRiODI0ZmIwNGI0M2E3Y2ZlM2Q1ZDQzOTcwYzkiLCJwIjoiaiJ9

Reply:
Okay, hab ich mir fast gedacht 👍